Sources/Remote Logging/Crash Logging/CrashLoggingDataProvider.swift

```diff
@@ -89,4 +90,8 @@ public extension CrashLoggingDataProvider {
     var enableCaptureFailedRequests: Bool {
         return false
     }
+
+    var eventSamplingRate: Double {
+        return 0.01
```

Comment:
From [the Sentry docs](https://docs.sentry.io/platforms/apple/guides/ios/configuration/sampling/#sampling-error-events): "The error sample rate defaults to 1, meaning all errors are sent to Sentry."

Setting this as `0.01` by default might result in clients that are not aware of this new configuration to lose 99% of their error events.

What do you think of keeping the same default as Sentry?

```suggestion
        return 1.0
```

Reply:
Sounds super, thank you Gio!!